### PR TITLE
fix(config-forms)

### PR DESCRIPTION
**Pull request by jalMogo**
This PR fixes some errors when importing the forms

### CODE REVIEW

File: src/flavors/columbia/config.json [hunk 1]
```diff
@@ -1706,7 +1706,7 @@
           },
           {
             "start_field_index": 62,
-            "end_field_index": 73,
+            "end_field_index": 71,
```
Comment (jalMogo) [inline src/flavors/columbia/config.json]: I was getting an "index out of bounds" error when parsing the `fields` array - it seems like we ignore this field, or silently fail, when this number is greater than the length of the array?

I double checked, and I think these new numbers are correct.

File: src/flavors/kittitas-vsp/config.json [hunk 2]
```diff
@@ -1194,7 +1194,7 @@
             "//": "=============== IRRIGATION SCHEDULING ========================",
             "name": "irrigation_scheduling",
             "type": "big_radio",
-            "prompt": "Irrigation scheduling – monitoring soil moisture and adjusting water applications accordingly",
+            "prompt": "Irrigation scheduling - monitoring soil moisture and adjusting water applications accordingly",
```
Comment (jalMogo) [inline src/flavors/kittitas-vsp/config.json]: These diffs are replacing the unicode characters with their ascii equivalents. Not a big deal.

File: src/flavors/columbia/config.json (hunk 1 above)
Comment (goldpbear) [reply src/flavors/columbia/config.json]: I think that's right-- the very last index is not enforced. As long as the final item visible is the submit button with the new indices it should be fine.

File: src/flavors/snohomish/config.json [hunk 3]
```diff
@@ -594,11 +594,11 @@
             ],
             "content": [
               {
-                "label": "Practice rotational grazing, never grazing grass below 3” deep",
```
Comment (goldpbear) [inline src/flavors/snohomish/config.json]: The ASCII changes seem fine except for these two-- the unicode typographer's quotes should be replaced with bunny ears (i.e. `"`).

Comment (goldpbear) [reply src/flavors/snohomish/config.json]: Or, alternatively, just write out the phrase: `...below three feet deep`.

Comment (jalMogo) [reply src/flavors/snohomish/config.json]: Ah, nice catch. I'll update it by spelling it out with `...below three inches deep`.

**Pull request merged**